service/routes.py

```diff
@@ -31,7 +31,23 @@ def index():
 
 @app.route("/recommendations", methods=["GET"])
 def list_recommendations():
-    pass
+    products = Recommendation.all()
+    results = [product.serialize() for product in products]
+    app.logger.info("Returning %d products", len(results))
+    return make_response(jsonify(results), status.HTTP_200_OK)
+    """Returns all of the recommendation"""
```

Comment:
Remove all codes after the return statement, include commented codes

Reply:
it is left for next spr

Reply:
Yeah keep it in your local, not remote

Reply:
i will remove that, but you know left it in local is very dangers.

next time pull command will have a conflict and may destroy that